datumaro/components/dataset.py

```diff
@@ -834,6 +881,16 @@ def flush_changes(self):
     @scoped
     def export(self, save_dir: str, format: Union[str, Type[Extractor]],
             **kwargs) -> None:
+        """
+        Saves the input dataset in some format.
+
+        Args:
```

Comment:
Documentation for `save_dir` is missing.

Reply:
Added